### PR TITLE
Use new version of ran which uses properties instead of tokens to sig…

### PR DESCRIPTION
**Pull request by anton-larsen-persequor**
…nify fields/columns on model/tables

### CODE REVIEW

File: valqueries-automapper/src/main/java/com/valqueries/automapper/CapturingHydrator.java [hunk 1]
```diff
@@ -39,7 +40,7 @@ public String getString(Token token) {
 	}
 
 	@Override
-	public Character getCharacter(Token token) {
+	public Character getCharacter(Property token) {
```
Comment (emanuel-persequor) [inline valqueries-automapper/src/main/java/com/valqueries/automapper/CapturingHydrator.java:43]: why is the `Property` parameter called "token" :-)

**Comment by andrepersequor** [comment]
@anton-larsen-persequor seems its missing the bumping version change

File: valqueries-automapper/src/main/java/com/valqueries/automapper/elements/SimpleElement.java [hunk 2]
```diff
@@ -24,7 +24,7 @@ public SimpleElement(ValqueriesQueryImpl<?> query, Property.PropertyValue<?> pro
 	}
 
 	public String queryString() {
-		return query.getTableAlias()+"."+dialect.escapeColumnOrTable(sqlNameFormatter.column(propertyValue.getProperty().getToken()))+" "+operator+" (:"+field+")";
+		return query.getTableAlias()+"."+dialect.column(propertyValue.getProperty().getToken())+" "+operator+" (:"+field+")";
```
Comment (andrepersequor) [inline valqueries-automapper/src/main/java/com/valqueries/automapper/elements/SimpleElement.java:27]:
```suggestion
		return query.getTableAlias()+"."+dialect.column(propertyValue.getProperty())+" "+operator+" (:"+field+")";
```
so it considers DbName into it

**Review by andrepersequor — APPROVED** [review]
lgtm and tests are passing

**Pull request merged**